doc/man3/OpenSSL_version.pod

```diff
@@ -17,11 +17,8 @@ OPENSSL_VERSION_NUMBER, OpenSSL_version_num, OPENSSL_info
  #define OPENSSL_VERSION_MAJOR  x
  #define OPENSSL_VERSION_MINOR  y
  #define OPENSSL_VERSION_PATCH  z
-
- /* The definitions here are typical release values */
```

Comment:
Oh!  Yeahok, that's something I didn't anticipate in my change.  I absolutely think that comments should be allowed and remain.